frontend/Components/Login.js

```diff
@@ -0,0 +1,210 @@
+import {
+    Text,
+    View,
+    StyleSheet,
+    Image,
+    TextInput,
+    Pressable,
+    Platform
+} from 'react-native';
+import React from 'react';
+
+
+export const Login = ({ navigation}) => {
+    return (
+        <View style={Styles.device}>
+            <View style={Styles.container}>
+                <Image 
+                    source={logoPath}
+                    style={Styles.logo}
+                /> 
+                <View style={Styles.titleContainer}>
+                    <Text 
+                        style={Styles.title}
+                        adjustsFontSizeToFit
+                    >
+                        Log in
+                    </Text>
+                </View>
```

Comment:
This could also be extracted to another component `Title` because it's used again in the signup form and also to avoid having too much nesting and unnecessary details in a single component.

In general, a component shouldn't be long, and you should be able to the understand the overall layout of a component at a glance without too much effort.

and btw you could access what is between the opening and closing tags by using `props.children`, so if we had `<Title>Log in</Title>` you could access it inside the Title component by using `{props.children}`